button.css

```diff
@@ -0,0 +1,98 @@
+.button {
+    background-color: rgb(183, 231, 183);
+    border: none;
+    padding: 16px 16px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 10px 2px;
+    transition-duration: 0.4s;
+    cursor: pointer;
+}
+
+.button1 {
+    background-color: rgb(39, 243, 12);
+    color: black;
+    border: 2px solid;
+}
+
+.button2 {
+    background-color: rgb(39, 235, 235);
+    color: black;
+    border: 2px solid #0f0f0e;
+    border-radius: 8%;
```

Comment:
Be consistent in color values. Either use rgb or use hex code. Avoid using direct color names